src/components/description/Description.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export default function Description(props) {
+  const { description, metadata } = props;
+
+  return (
+    <div className="description">
+      <p>{description}</p>
+
+      {Object.entries(metadata).map(([layerName, layerMetadata]) => (
+        <details key={layerName}>
+          <summary>{layerName}</summary>
+          <div className="metadata-container">
+            <table>
+              <tbody>
+                {Object.entries(layerMetadata)
+                  .map(([key, value]) => (
+                    <tr key={key}>
+                      <th title={key}>{key}</th>
+                      <td title={value}>{value}</td>
```

Comment:
Do you get ARIA warnings without the title? If it is necessary, perhaps:
```suggestion
                      <th title="key">{key}</th>
                      <td title="value">{value}</td>
```

Reply:
I intentionally added the title attributes since I added CSS to allow the table cell text to be truncated with ellipses when the table is too narrow. This way, when the text is not entirely visible, hovering will trigger the browser tooltip with all of the text.